### ISSUE TITLE
error inside of a git repo folder

### ISSUE THREAD

**Issue by azinsharaf** [ogham/exa#1132]
running `lla` inside of a `Git` repository folder returns this error.

```
exa: Options --git and --git-ignore can't be used because `git` feature was disabled in this build of exa
```

**Comment by daviessm** [comment]
What is `lla` an alias for?

**Comment by azinsharaf** [comment]
good question. It is an alias created by `https://github.com/gazorby/fish-exa`.
it is : `alias lla 'exa_git $EXA_LA_OPTIONS'`
```
$ echo $EXA_LA_OPTIONS
--all --binary
```
my goal is to use it instead of `ls -lah`

**Comment by daviessm** [comment]
And what is `exa_git`?

**Comment by mschilli87** [comment]
@daviessm:
> And what is `exa_git`?

AFAICT [basically `exa --git`](https://github.com/gazorby/fish-exa/blob/master/functions/exa_git.fish).

**Comment by daviessm** [comment]
So this would mean that the version of `exa` you have installed wasn't compiled with the `git` feature. I thought most distributions enabled `git` with `exa`, if you used a package manager to install `exa`, which distribution are you using?

**Comment by eggbean** [comment]
exa 9.0 is the current version on Debian 11.5 and has no exa support. Not sure about Debian testing, which is on v10. I've been having to compile myself.

@azinsharaf Use the binaries in the release zip files or compile yourself if you are using a Debian-based distro.

**Comment by azinsharaf** [comment]
I am using exa/jammy 0.10.1 on Ubuntu 22.04.

**Comment by eggbean** [comment]
Yes, well Ubuntu is based on Debian and uses the same exa. With the popularity of Ubuntu, most exa users probably don't have the git functionality.

**Comment by eggbean** [comment]
If you extract the binary from the zip file and put it in `/usr/local/bin`, it will fix your problem.

**Comment by azinsharaf** [comment]
the existing binaries didn't work on my machine (probably because i am using Ubuntu 22.04 with WSL.) I got the error that the application cannot run on this operating system.
Anyway, I built it from source and now it works fine even in a git repo folder.
Thank you all.
